common/Fabric-Install-Utilities.psm1

```diff
@@ -190,6 +190,29 @@ function Test-Prerequisite($appName, $minVersion)
     }
 }
 
+function Test-Prerequisite-Exact($appName, $supportedVersion)
+{
+    $installedAppResults = Get-InstalledApps | where {$_.DisplayName -like $appName}
+    if($installedAppResults -eq $null){
+        return $false;
+    }
+
+    if($supportedVersion -eq $null)
+    {
+        return $true;
+    }
+
+    $minVersionAsSystemVersion = [System.Version]$supportedVersion
```

Comment:
should we change this variable to represent more what it is, perhaps supportedVersionAsSystemVersion?